src/test/java/io/kontur/disasterninja/service/layers/providers/LayersApiProviderIT.java

```diff
@@ -100,7 +100,7 @@ public void getLayers() throws JsonProcessingException, ExecutionException, Inte
         assertFalse(layer.isBoundaryRequiredForRetrieval());
 
         Layer layer11 = layers.get(11);
-        assertEquals("KLA__test12", layer11.getId());
+        assertEquals("test12", layer11.getId());
```

Comment:
Hi! do we need this test layer?

Reply:
yes, this is the last layer in response. Is used to check pagination

Reply:
There are 12 layers in the test response and this is the 12th of them. That is why it has this id. This is not a real layer. And it's made just for test purposes.

Reply:
it looks a bit weird with off-by-one naming (test12 but layer11), otherwise seems ok.